src/app/(login)/sign/page.tsx

```
@@ -0,0 +1,147 @@
+
+          <div>
+            <label className="mb-1 block text-sm font-bold text-blue-950">아이디 (이메일)</label>
+            <input
+              type="email"
+              placeholder="아이디 입력"
+              className="w-full border border-gray-200 bg-gray-100 px-4 py-3 text-sm outline-none focus:border-blue-500"
+            />
+          </div>
+
+          <div>
+            <label className="mb-1 block text-sm font-bold text-blue-950">비밀번호</label>
```

Biome (2.4.4)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)